hoverkite-protocol/src/response.rs

```diff
@@ -0,0 +1,189 @@
+use crate::Side;
+use std::collections::VecDeque;
+use std::convert::TryInto;
+
+#[derive(Clone, Debug, Eq, PartialEq)]
+pub struct Response {
+    pub side: Side,
+    pub response: SideResponse,
+}
+
+#[derive(Clone, Debug, Eq, PartialEq)]
+pub enum SideResponse {
+    Log(String),
+    Position(i64),
```

Comment:
(fine like this for now): it's slightly questionable that Position messages are a Response, since they happen with nobody asking for them (I like the name Report).